src/main.js

```diff
@@ -104,7 +105,7 @@ export default class Auth {
 	}
 
 	/**
-	 * Makes a post request to a specific endpoint and returns the response.
+	 * Makes post request to a specific endpoint and return the response.
```

Comment:
Here too it should be as it was.
If you are checking this with grammarly or something similar it will think that “post” is a verb, but it isn’t, so it doesn’t make sense to remove “a”. And it should also remain “returns”.